js/webpack_get_defaults.js

```diff
@@ -80,11 +81,14 @@ module.exports = function (basePath) {
     })
   }
   if (config.build.type === enums.buildTypes.APPLICATION) {
-    plugins.push(new HtmlPlugin({
-      inject: true,
-      filename: 'index.html',
-      template: path.join(config.build.src_dir, 'index.html'),
-    }))
+    const indexHtmlPath = path.join(config.build.src_dir, 'index.html')
+    if (fs.existsSync(indexHtmlPath)) {
```

Comment:
Je suppose... on pourrait crasher mais avec une erreur un peu mieux foutue
que celle que vomit le plugin en crashant^^

Mais une application de 'composants' pourrait être distribuée sans index,
no ?
On May 27, 2016 14:17, "Raphaël Boucher" <notifications@github.com> wrote:

> In js/webpack_get_defaults.js
> <https://github.com/Polyconseil/systematic/pull/14#discussion_r64896100>:
>
> > @@ -80,11 +81,14 @@ module.exports = function (basePath) {
> >      })
> >    }
> >    if (config.build.type === enums.buildTypes.APPLICATION) {
> > -    plugins.push(new HtmlPlugin({
> > -      inject: true,
> > -      filename: 'index.html',
> > -      template: path.join(config.build.src_dir, 'index.html'),
> > -    }))
> > +    const indexHtmlPath = path.join(config.build.src_dir, 'index.html')
> > +    if (fs.existsSync(indexHtmlPath)) {
>
> Pourquoi ne pas crasher à ce moment? On peut faire des apps sans
> index.html?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Polyconseil/systematic/pull/14/files/1611e941a979d7841ecae013a676766adffda650..eac4fa7909178bf2ef3eab644f9b2d265b0807c8#r64896100>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAFaKt_wCAPNSIRpm4iqDoau5FVqn6q_ks5qFuDOgaJpZM4IntYR>
> .
>